src/sentry/incidents/utils/metric_issue_poc.py

```diff
@@ -61,19 +67,35 @@ def construct_title(alert_rule: AlertRule) -> str:
     else:
         higher_or_lower = "less than" if alert_rule.comparison_delta else "below"
 
+    label = INCIDENT_STATUS[IncidentStatus(status)].lower()
+
     # Format the time window for the threshold
     time_window = alert_rule.snuba_query.time_window // 60
-    title = f"{agg_text} in the last {format_duration_idiomatic(time_window)} {higher_or_lower}"
+    title = f"{label.upper()}: {aggregate}"
```

Comment:
IMO: Uppercase is a bit shouty for the title, especially since in the UI it'll be the biggest text. Maybe just title-case? `Critical:`, `Warning:` (unless design wants this in which case feel free to dismiss)